cli/src/account.rs

```diff
@@ -60,15 +70,9 @@ pub async fn account_prompt(wallet: &Wallet, mut account: Account) -> Result<(),
         .edit_mode(rustyline::EditMode::Emacs)
         .build();
 
-    // Panic: should never happen
+    // Panic: should never happen since it's all in-memory
     let mut rl = Editor::with_history(config, MemHistory::with_config(config)).unwrap();
```

Comment:
This failure can be from an error accessing the terminal, so the comment isn't quite accurate. Maybe we should create an error variant for this instead

Reply:
oh damn, you're right ...

Reply:
should we just map it to an IO error?

Reply:
If that's possible, then sure

Reply:
changed my mind and added an error variant as you proposed